src/jsifier.js

```diff
@@ -138,7 +138,7 @@ function runJSify() {
         // body in an inner function.
         newArgs = newArgs.join(',');
         return `\
-function ${name}(${args}) {
+function(${args}) {
```

Comment:
Isn't it better for stack traces to have the name?

Reply:
They will all have names in the final output.   The final output looks like either:

```
function <Name>(...) { .. };`
```

or

```
var <Name> = (...) => { .. }
```

Both of these forms are enough for v8 to add names to the stack traces.

The name gets added by jsifier.js down around line 444 where it says `// name the function; overwrite if it's already named `.

Reply:
Since the names get added by jsisfier.js I could potentially to a preparatory PR to avoid naming the functions in these places unnecessarily.

Reply:
Split out #19551 which has now landed so this is now longer part of this PR.